analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -61,33 +59,18 @@ def get_warnings(env=None):
     if not diagtool_bin:
         return []
 
-    command = [diagtool_bin, 'tree']
     try:
         result = subprocess.check_output(
-            command,
+            [diagtool_bin, 'tree'],
             env=env,
             universal_newlines=True,
             encoding="utf-8",
             errors="ignore")
-        return parse_warnings(result)
+        return result.split()
     except (subprocess.CalledProcessError, OSError):
         return []
 
 
-def parse_warnings(diagtool_output):
```

Comment:
Why did you remove this function? Without this it will not skip some line from the diagtool output:
```

GREEN = enabled by default

-W
  -Wextra
```
This function also removed the white spaces.

Reply:
The first line goes to the standard error, so it is not in the standard output what we parse. The extra spaces are eliminated by `split()` function.